src/components/approval-type/ApprovalType.vue

```diff
@@ -117,148 +114,147 @@ export default class ApprovalType extends Vue {
     courtNumRef: FormIF
   }
 
-  /** Draft court order number. */
+  /** The initial court order number. */
   @Prop({ default: '' }) readonly courtOrderNumber!: string
 
-  /** Whether approved by the registrar. */
+  /** Whether restoration was approved by the registrar. */
   @Prop({ default: false }) readonly approvedByRegistrar!: boolean
 
-  /** filing name used in radio options. */
-  @Prop({ default: 'restoration' }) readonly filingType!: string
+  /** Whether restoration was approved by court order. */
+  @Prop({ default: false }) readonly approvedByCourtOrder!: boolean
 
-  /** Show only the court order option; remove via registrar option. */
-  @Prop({ default: false }) readonly isCourtOrderOnly!: boolean
+  /** Filing name used in radio options. */
+  @Prop({ default: 'restoration' }) readonly filingType!: string
 
-  /** Draft notice date. */
+  /** The initial date the notice of the application for restoration was published in the BC Gazette. */
   @Prop({ default: '' }) readonly noticeDate!: string
 
-  /** Draft application date. */
+  /** The initial date the application for restoration was mailed to the company. */
   @Prop({ default: '' }) readonly applicationDate!: string
 
   /** Whether this section is invalid. */
   @Prop({ default: false }) readonly invalidSection!: boolean
 
-  /** Whether the display of court order section is a radio button. */
+  /** Whether to show the court order radio button. */
   @Prop({ default: true }) readonly isCourtOrderRadio!: boolean
 
+  /** Whether validation is enabled. */
+  @Prop({ default: false }) readonly validate!: boolean
```

Comment:
Question about this prop. In my head, I'm thinking we never want it to be false. Why would we want to use the component with no validation enabled?

Reply:
In its initial state, we don't want to show validation errors. Once the user does something -- in this case, try to go to the Review and Confirm page -- then we check for and show errors.

So, with this prop False, this component is always valid. With this prop True then it depends what data has been entered (ie, the rules are applied).

Reply:
Ahhh, that's brilliant. OK, makes sense, thanks! Resolving.